R/geom_fit_text.R

```diff
@@ -92,6 +92,8 @@
 #' that aesthetics specifying the box must be provided. See Details.
 #' @param data,stat,position,na.rm,show.legend,inherit.aes,... Standard geom
 #' arguments as for `ggplot2::geom_text()`.
+#' @param flip If `TRUE` text will be rotated when it would otherwise be
```

Comment:
Suggest 'flipped 180 degrees' or 'flipped the right way up' instead of 'rotated'